data/json/recipes/basecamps/recipe_groups.json

```diff
@@ -201,23 +201,23 @@
     "id": "saltworks_recipes_2",
     "building_type": "COOK",
     "recipes": [
-      { "id": "brew_fruit_wine", "description": " Brew: Fruit Wine Must" },
-      { "id": "brew_mead", "description": " Brew: Spiced Wine Must" },
-      { "id": "brew_dandelion_wine", "description": " Brew: Dandelion Wine Must" },
-      { "id": "brew_burdock_wine", "description": " Brew: Burdock Wine Must" },
-      { "id": "brew_pine_wine", "description": " Brew: Pine Wine Must" },
-      { "id": "brew_hb_beer", "description": " Brew: Beer" }
+      { "id": "brew_fruit_wine", "description": " WIP Brew: Fruit Wine Must" },
```

Comment:
Do these still need to be renamed as WIP or does the solution work as expected?   It looks good from what I saw of it in your screenshots.